### ISSUE TITLE
OpenMPI v3.x: PMIX ERROR: UNPACK-PAST-END

### ISSUE THREAD

**Issue by ericch1** [open-mpi/ompi#3416]
Hi,

I am testing ompi/v3.x branch.

Installation logs:

http://www.giref.ulaval.ca/~cmpgiref/ompi_3.x/2017.04.26.08h32m32s_config.log

ompi_info -all:

http://www.giref.ulaval.ca/~cmpgiref/ompi_3.x/2017.04.26.08h32m32s_ompi_info_all.txt

I am running my test suite *locally* on a OpenSuse 13.2, Intel(R) Core(TM) i7-6700K CPU @ 4.00GHz

I have 2 sequential tests (ie *not* launched whi mpirun) that *hang* very very soon at startup:

first test:
```
[zorg:04089] [[INVALID],INVALID] plm:rsh_lookup on agent ssh : rsh path NULL
[zorg:04089] plm:base:set_hnp_name: initial bias 4089 nodename hash 810220270
[zorg:04089] plm:base:set_hnp_name: final jobfam 50525
[zorg:04089] [[50525,0],0] plm:rsh_setup on agent ssh : rsh path NULL
[zorg:04089] [[50525,0],0] plm:base:receive start comm
[zorg:04089] [[50525,0],0] plm:base:launch [50525,1] registered
[zorg:04089] [[50525,0],0] plm:base:launch job [50525,1] is not a dynamic spawn
[zorg:04087] PMIX ERROR: UNPACK-PAST-END in file client/pmix_client.c at line 113
[zorg:04089] [[50525,0],0] plm:base:receive stop comm
```

second test:
```
[zorg:02486] [[INVALID],INVALID] plm:rsh_lookup on agent ssh : rsh path NULL
[zorg:02486] plm:base:set_hnp_name: initial bias 2486 nodename hash 810220270
[zorg:02486] plm:base:set_hnp_name: final jobfam 49938
[zorg:02486] [[49938,0],0] plm:rsh_setup on agent ssh : rsh path NULL
[zorg:02486] [[49938,0],0] plm:base:receive start comm
[zorg:02485] PMIX ERROR: UNPACK-PAST-END in file client/pmix_client.c at line 113
[zorg:02486] [[49938,0],0] plm:base:launch [49938,1] registered
[zorg:02486] [[49938,0],0] plm:base:launch job [49938,1] is not a dynamic spawn
[zorg:02486] [[49938,0],0] plm:base:receive stop comm
```
the others (about 480) are successful...

Is it I too early for reporting bugs for 3.x ?

Thanks,

Eric

**Comment by rhc54** [comment]
Just to be clear - these are singletons - i.e., you started them with ./a.out instead of mpirun ./a.out?

There are a bunch of changes slated for the 3.x branch that have backed up, so I'm not too concerned about such things right now.

**Comment by ericch1** [comment]
Yes, singletons they are.

Ok, I wanted to test 3.0 when I saw it was planned for june 15, but I can't find where I saw this now... ???

Thanks anyway!

Eric

**Comment by bwbarrett** [comment]
It's still targeted for June, so updated this bug with the right tags.  Thanks for the bug report.

**Comment by rhc54** [comment]
We actually do appreciate the testing - I've been tied up this month, but have set May aside for resolving problems in the 3.x branch.

**Comment by rhc54** [comment]
Fixed and included in #3462

**Comment by ericch1** [comment]
Thanks, everything is working fine now! :)
Eric
